src/vs/server/entry.ts

```diff
@@ -68,17 +44,15 @@ export async function main(args: ServerArgs) {
 		let query = new URLSearchParams();
 
 		if (request.url) {
+			// TODO use `socket.remoteAddress`
```

Comment:
Not sure if this is our todo or theres? (entry is our file right?)
```suggestion
			// TODO@coder use `socket.remoteAddress`
```